infra/scripts/test_android_runtime_release.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+[[ "${BASH_SOURCE[0]}" != "${0}" ]] && echo "Please don't source ${BASH_SOURCE[0]}, execute it" && return
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_PATH="$CURRENT_PATH/../../"
+
+# Model download server setting
+if [[ -z "${MODELFILE_SERVER}" ]]; then
+  echo "[ERROR] Model file server is not set"
+  echo "        Need to download model file for test"
+  exit 1
+else
+  echo "Model Server: ${MODELFILE_SERVER}"
+fi
+
+$ROOT_PATH/Product/out/test/models/run_test.sh --download=on --run=off
+$ROOT_PATH/Product/out/test/models/run_test.sh --download=on --run=off \
+  --configdir=$ROOT_PATH/Product/out/test/models/nnfw_api_gtest \
+  --cachedir=$ROOT_PATH/Product/out/unittest_standalone/nnfw_api_gtest_models
+
+N=`adb devices 2>/dev/null | wc -l`
+
+# exit if no device found
+if [[ $N -le 2 ]]; then
+    echo "No device found."
+    exit 1;
+fi
+
+NUM_DEV=$(($N-2))
+echo "device list"
+DEVICE_LIST=`adb devices 2>/dev/null`
+echo "$DEVICE_LIST" | tail -n"$NUM_DEV"
+
+if [ -z "$SERIAL" ]; then
+    SERIAL=`echo "$DEVICE_LIST" | tail -n1 | awk '{print $1}'`
+fi
+ADB_CMD="adb -s $SERIAL "
+
+# root on, remount as rw
+$ADB_CMD root on
+$ADB_CMD shell mount -o rw,remount /
+
+$ADB_CMD shell rm -rf /data/local/tmp
```

Comment:
This will /data/local/tmp/ folder ? This folder may contain pre installed binaries by users.

Reply:
Oh, I don't know that... then I think I should change the binary installation folder on others. Thanks for pointing out!